### DISCUSSION TITLE
Annualized LCIA by using annualized LCI and background scenarios

### DISCUSSION THREAD

**Discussion by Paul-Robineau** [LCA-ActivityBrowser/activity-browser#1187] [Q&A]
Hello there!

I am looking for the optimal way to use AB when computing the annualized LCIA of an annualized LCI, depending on different scenarios retrieved through the ScenarioLink plugin.

Let's say I intially have a table: 
- 4 lines, each representing a different material
- 15 rows describing transportation amounts by road for the 2026-2040 time period

| Material | 2026 | 2027 | 2028 | 2029 | 2030 | 2031 | 2032 | 2033 | 2034 | 2035 | 2036 | 2037 | 2038 | 2039  | 2040 |
|----------|------|------|------|------|------|------|------|------|------|------|------|------|------|-------|------|
| A        | 0    | 0    | 0    | 69   | 69   | 63   | 0    | 0    | 0    | 0    | 0    | 0    | 0    | 0     | 0    |
| B        | 774  | 982  | 619  | 471  | 610  | 681  | 761  | 512  | 421  | 204  | 105  | 70   | 0    | 0     | 0    |
| C        | 1765 | 2171 | 1118 | 586  | 342  | 565  | 760  | 1053 | 1185 | 1075 | 636  | 614  | 139  | 139   | 73   |
| D        | 0    | 0    | 0    | 0    | 0    | 0    | 0    | 0    | 0    | 0    | 1523 | 4996 | 7469 | 18285 | 2735 |

For each line, I want AB to do: 
- association of transportation amounts of a year => the activity in the relevant version of futurized ecoinvent
- calculation of the impact for each of these years, still being able to distinguish years & materials 
(I can do the total afterwards, if needed, in a classical spreadsheet).

I am pretty sure there is a way of doing this "simply" by doing a Scenario LCA combining an elementary flows foreground scenario file and the background superstructure scenario file – in the same fashion as demonstrated in the 6th tutorial video. 
That being said, I don't know how to construct this foreground file so that the transported quantities are "tagged" with the appropriate year and then associated by AB with the adequate (nearest in time) futurized ecoinvent of the superstructure during the LCIA calculations.

Does someone here already tried to do this or have answers/advices? 

_Appendix Note: A first method could be to manually do it, by having as much activities as there is materials and and years to account, and for each activity use the appropriate futurized ecoinvent database. This means creating 4*15=60 activities, and each time selecting the proper (nearest in time) future ecoinvent individually. As it does not seem possible to individually select the relevant year for the activity when using the superstructure database (I may be wrong on that, please tell me!), it would also means creating each future ecoinvent database from 2025 to 2040 individually. This would clearly be unefficient, to say the least._

**Comment by marc-vdm** [comment]
@romainsacchi, do you think a good approach here would be to use a [community scenario](https://github.com/premise-community-scenarios)?

**Reply by Paul-Robineau** [reply]
I looked a bit at community scenarios: Even if premise is primarly designed to modify background scenarios, I guess it could be used to modify a foreground database and annualize it, but I not able (yet) to say if the time dynamic correspondence between background and foreground data would be done during computation in AB.

It seems to me that @romainsacchi and colleagues (notably @tapajyotighosh10) developped the [LiAISON framework](https://pubs.acs.org/doi/10.1021/acs.est.2c04246) specifically to adress combination between dynamic LCA (understood here as "with foreground temporal variation") and prospective LCA (resp. "background variation"). I guess that framework could be a building block for future implementation in an GUI-based (more accessible) use in AB.

In the mean time, I'll directly contact them both to discuss use of LiAISON to answer my problem, as I did not found the dedicated repository.
